lib/snapshot/BUILD.gn

```diff
@@ -8,13 +8,9 @@ import("//flutter/common/config.gni")
 import("//flutter/lib/ui/dart_ui.gni")
 import("//third_party/dart/utils/compile_platform.gni")
 
-bindings_output_dir = "$root_gen_dir/sky/bindings"
-
-copy("generate_dart_ui") {
+# TODO(dnfield): Remove this when recipes are updated to not use it.
```

Comment:
which recipes are this refering to? This also needs a link

Reply:
The LUCI recipes

Reply:
https://flutter-review.googlesource.com/c/recipes/+/15420

Reply:
Let's include the link in the todo so we know when to clean up this once it merges

Reply:
Done